#!/usr/bin/env python

PACKAGE = "spatial_audio_demo"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add('rhythm_ratio', double_t, 0, 'Rhythm ratio', 1.0, 0, 10.0)
gen.add('volume', double_t, 0, 'Volume', 1.0, 0, 10.0)

exit(gen.generate(PACKAGE, 'sound_navigator', 'BeaconSoundPublisher'))
